src/ImageSharp/Memory/DiscontiguousBuffers/MemoryGroup{T}.Consumed.cs

```diff
@@ -48,6 +48,6 @@ IEnumerator<Memory<T>> IEnumerable<Memory<T>>.GetEnumerator()
             return ((IList<Memory<T>>)this.source).GetEnumerator();
         }
 
-        public override void Dispose() => this.View.Invalidate();
+        public override void Dispose() => this.View?.Invalidate();
```

Comment:
Now that `View` is non-nullable there is no need for `?`. Same the for changes below.